UPGRADE.md

```diff
@@ -2,6 +2,12 @@
 
 This file contains instructions to upgrade to a new release of Gregorio.
 
+## Unreleased
+
+### Empty first syllable hyphen
+
+Prior to version 4.0, GregorioTeX would put a hyphen below an empty first syllable (which happens when the first syllable consists only of the initial).  The spacing algorithm was improved in 4.0, but that eliminates the hyphen if there is not enough space between the note of the first syllable and the next syllable.  The `\gresetemptyfirstsyllablehyphen` command controls this behavior.  It defaults to `force`, which restores the pre-4.0 behavior, but may be changed to `auto` (i.e., `\gresetemptyfirstsyllablehyphen{auto}`) if you prefer the space-based behavior.
```

Comment:
I'm a little confused from your statement.  Are you saying that we *should* do this change in 4.0 or leave it here in develop?  If you think the scope of these changes is not "too much" for this late in the 4.0 cycle, then I can reopen the pull request against release-4.0.

My only concern with doing it this late in 4.0 is code quality.  Of course, programmers never think they write buggy code, but experience often proves otherwise...

If this is not what you mean, can you clarify?